### PR TITLE
Set `node-version` to 18.15.0

### LINKED ISSUE #1242: Node github actions failing on `main`
https://github.com/statechannels/go-nitro/actions/runs/4802488125/jobs/8546007326

### PR DESCRIPTION
**Pull request by geoknee**
https://github.com/NomicFoundation/hardhat/issues/3877

Closes #1242

### CODE REVIEW

File: nitro-protocol/readme.md [hunk 1]
```diff
@@ -91,6 +91,6 @@ After deployment, run
 ETHERSCAN_API_KEY=<a-secret> INFURA_TOKEN=<another-secret> yarn hardhat --network rinkeby verify <DeployedContractAddress> 'ConstructorArgs'
 ```
 
-for each contract you wish to verify. Swap rinkeby for mainnet as appropriate.
+for each contract you wish to verify. Swap rinkeby for mainnet as appropriate!
```
Comment (NiloCK) [inline nitro-protocol/readme.md:94]: Is this:
- unrelated drive-by
- accidental
- intended to be "some changeset" so that the gh action runs
?

**Review by NiloCK — APPROVED** [review]
Looks good. Were `main` CI failures intermittent?

**Comment by geoknee** [comment]
> Looks good. Were `main` CI failures intermittent?

Somewhat, yes.
